pyproject.toml

```diff
@@ -15,6 +15,7 @@ crewai = ">=0.28.8,<0.31.0"
 langchain-anthropic = "^0.1.11"
 langchain-groq = "^0.1.3"
 ollama = "^0.2.0"
+jupyter = "^1.0.0"
```

Comment:
`jupyter` is not required by the package to run - it is to be used by developers - and should be included in `...dev.dependencies` below (to prevent it from being included in the packages shipped with the extension)